public/scripts/githubToken.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
```

Comment:
oops, you pushed up your github token! This should always be gitignored so no one can mess with your repos